features/netsocket/nsapi_ppp.h

```diff
@@ -36,10 +36,11 @@ NetworkStack *nsapi_ppp_get_stack();
  *  @param status_cb    Optional, user provided callback for connection status
  *  @param uname        Optional, username for the connection
  *  @param pwd          Optional, password for the connection
+ *  @param stack        Optional, stack for the connection
  *
  *  @return             0 on success, negative error code on failure
  */
-nsapi_error_t nsapi_ppp_connect(FileHandle *stream, Callback<void(nsapi_error_t)> status_cb=0, const char *uname=0, const char *pwd=0);
+nsapi_error_t nsapi_ppp_connect(FileHandle *stream, Callback<void(nsapi_error_t)> status_cb=0, const char *uname=0, const char *pwd=0, const nsapi_ip_stack_t stack=IPV4_STACK);
```

Comment:
Why default to IPv4 ? Isn't the whole point to have the stack dual configurable and choose at runtime which one is available ?